pkgs/by-name/wc/wcurl/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  curl,
+  fetchFromGitHub,
+  makeWrapper,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wcurl";
+  version = "2024.07.10";
+
+  src = fetchFromGitHub {
+    owner = "Debian";
+    repo = "wcurl";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-FYkG74uoXFNYT7tszDcdCPQCEG3ePOFBUgIUYpsAzb8=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+
+  postPatch = ''
+    # prevent .wcurl-wrapped from appearing in the help output
+    sed -i -e 's/"$(basename "$0")"/wcurl/' wcurl
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D wcurl -t $out/bin
+    wrapProgram $out/bin/wcurl --prefix PATH : ${lib.makeBinPath [ curl ]}
```

Comment:
It only appeared here https://salsa.debian.org/debian/wcurl/-/blob/main/wcurl?ref_type=heads#L117. Maybe just `substituteInPlace` it in `postPatch`?